proposals/0017-priced-compute-units.md

```diff
@@ -0,0 +1,34 @@
+---
+simd: '0017'
+title: Priced Compute Units
+authors:
+  - Anoushk Kharangate (SuperteamDAO)
+  - 7Layer (Overclock Validator)
+  - Dubbelosix (Overclock Validator)
+category: Fees
+type: Fees
+status: Draft
+created: 2023-01-05
+---
+
+## Problem
+According to the discussion on the following issue:
+https://github.com/solana-labs/solana/issues/29492
+
+Recently blocks have been reaching their compute unit limit of 48 million CU very frequently. The concerning part is most of these transactions
+are requesting extra compute units without actually needing them; most of these being mev traders. The reason this happens is because
+using extra compute units comes at no extra cost unlike something like rent but validators have to bear the load and aren't compensated for it
+hence a new economic model is required for using compute that disincentivizes extra CU unless absolutely necessary.
+
+## Solution
+Introduce a new price and economic model for compute units. It's important to note that the new model might not want to make it more expensive to vote.
+
+A good way to normalize and do this would be charge a fee per compute unit of 5 x 10<sup>-6</sup> SOL/(vote CU) per CU<br/>
+Example: Txn A requires 10000 CU -> Cost of txn is 10000 x (5 x 10<sup>-6</sup> SOL)/2100 CU  (2100 is the vote CU)
```

Comment:
Add to end: Note that this normalization no longer includes a separate signature based fee which is outside of compute unit calculations. It may make sense to keep a separate signature fee if there is a dos vector here.